README.md

```
@@ -1,63 +1,146 @@
-* Customizable `minuteInterval` attribute
-
-### Platform Support & Theming
-
-* Fully support Web and Desktop platforms (using wheel views)
-* Built-in themes
-* Enhanced tablet view
-
-## Getting Started
-
-To add bottom picker to your project add this line to your pubspec.yaml file
+### Core Picker Types
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)